README.md

```diff
@@ -542,6 +543,15 @@ This function returns a bool indicating whether or not a window `win` exists on
 Optional arguments:
 - `env`: Environment to search for the window in. Default is `None`.
 
+#### vis.get_window_data
+This function returns the window data for the given window. Returns data for all windows in an env if win is None.
+
+Arguments:
+- `env`: Environment to search for the window in.
+- `win`: Window to return data for. Set to `None` to retrieve all the windows in an environment.
+
+*Note:* This function is python-only and doesn't exist in the Lua version.
```

Comment:
Should it not be easy to expose a wrapper of `get_window_data()` in the Lua client, also, or am I missing something?

Reply:
It wouldn't be, though I wasn't quite sure what the use case would be in Lua. I could add this before I push if you think it'd be useful